cukinia

```diff
@@ -27,6 +28,11 @@ GENERIC_FUNCTS='cmd
 		kversion
 		listen4
 		mount
+		network_interface_dhcp_ipv4
+		network_interface_dhcp_ipv6
+		network_interface_global_ipv6
```

Comment:
Why does ipv6 have a `global` scope and doesn't have the any scope option? Similarly, ipv4 only has the any scope option but no `global` option.

IMO, we should stick to "any scope" for both ipv4 and ipv6.

Reply:
In IPv4 there is not concept of link local but in case of IPv6 we will always have link-local IP . So no point of checking link-local and that's why we are only checking global. Yes we need to add more functions(In future) to capture complete IPv6 addressing.

Reply:
ok, make sense, thank you.